data_collection/web/guides.rst

```diff
@@ -10,3 +13,46 @@ Downloads and Outlinks
 ----------------------
 
 Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop p
+
+Site search
+-----------
+
+Site search tracking gives you insight into how users interact with the search
+engine on your website - what they search for and how many results they get
+back.
+
+Our data collecting and processing pipeline automatically converts page views
+into site search events if the URL contains site search query parameters:
+``q``, ``query``, ``s``, ``search``, ``searchword`` and ``keyword``.  You can
+customize these parameters in on website settings page. Site search events can
+also be tracked manually by calling :ref:`trackSiteSearch<js-api-trackSiteSearch>`
+method. It allows to specify not only the keyword and category, but also
+the number of results and additional custom dimensions.
+
+:ref:`trackSiteSearch<js-api-trackSiteSearch>` accepts the following parameters:
+
+* **keyword** - what term someone looked for
+* **category** (optional) - which category the search was in
+* **results** (optional) - how many search results were returned
+* **dimensions** (optional) - custom dimensions to send along the site search
+
+It is used like this::
+
+    _paq.push(["trackSiteSearch", "les paul", "electric guitars", 5, { dimension10: "amber" }]);
+
+In this case, we track site search with keyword *les paul*, category *electric
+guitars*, *5* search results and custom dimension *10* with value *amber*.
+
+The optional parameters might be skipped or replaced with ``undefined`` to
+indicate no value. ::
+
+    _paq.push(["trackSiteSearch", "playstation"]); // only keyword provided
+    _paq.push(["trackSiteSearch", "playstation", "consoles"]); // only keyword and category provided
+    _paq.push(["trackSiteSearch", "playstation", undefined, 5]); // only keyword and results count provided
+
+.. warning::
+
+    If you can't or don't want to rely on automatic site search detection from
+    URL parameters, call ``trackSiteSearch`` method instead of ``trackPageView``
+    on search results page. Using both methods might result in duplication
```

Comment:
Some article should be there - I'm just not sure if `a` or `the` - `a` seems more appropriate (to me at least).
```suggestion
    on search results page. Using both methods might result in a duplication
```